### PR TITLE
Add typings for meow

### PR DESCRIPTION
**Pull request by KnisterPeter**
Please fill in this template.

- [x] Prefer to make your PR against the `types-2.0` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes).

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Run `tsc` without errors.
- [x] Include the required [files](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package) and header. Base these on the README, *not* on an existing project.

### CODE REVIEW

**Comment by KnisterPeter** [comment]
As requested by meow maintainers here sindresorhus/meow#55

File: meow/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "strictNullChecks": false,
```
Comment (andy-ms) [inline meow/tsconfig.json]: Why not?

File: meow/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Type definitions for meow 3.6
+// Project: https://github.com/sindresorhus/meow
+// Definitions by: KnisterPeter <https://github.com/KnisterPeter>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="minimist" />
```
Comment (andy-ms) [inline meow/index.d.ts]: Shouldn't need a reference if you have an import.

**Comment by KnisterPeter** [comment]
:+1: thanks

**Pull request merged**